### PR TITLE
Translated docs/actions.md

### PR DESCRIPTION
**Pull request by gauravgupta45**
Completed the translation of `docs/actions.md`. Merge it once you're satisfied :)

### CODE REVIEW

**Review by SamuelAlev — CHANGES_REQUESTED** [review]
Please **do not** translate the keys.

**Comment by gauravgupta45** [comment]
Is that fine now?

**Comment by gauravgupta45** [comment]
Check if it is fine now.

File: docs/docs/actions.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
 title: Actions
-description: Documentation on actions and how they help you manipulate state within Gatsby
+description: Documentation sur les actions et comment elles vous aident à manipuler l'état dans Gatsby
 jsdoc:
   - "gatsby/src/redux/actions/public.js"
   - "gatsby/src/redux/actions/restricted.js"
-contentsHeading: Functions
+contentsHeading: les fonctions
```
Comment (SamuelAlev) [inline docs/docs/actions.md]:
```suggestion
contentsHeading: Les fonctions
```

**Review by SamuelAlev — CHANGES_REQUESTED** [review]

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**